src/proto.c

```diff
@@ -73,21 +67,6 @@ static inline bool is_token_valid(const char *token)
 	return strlen(token) == KNOT_PROTOCOL_TOKEN_LEN;
 }
 
-static struct proto_ops *get_proto_ops(const char *protocol_name)
-{
-	int i;
-	struct proto_ops *selected_protocol = NULL;
-
-	for (i = 0; proto_ops[i]; i++) {
-		if (strcmp(protocol_name, proto_ops[i]->name) != 0)
-			continue;
-
-		selected_protocol = proto_ops[i];
-	}
-
-	return selected_protocol;
-}
-
```

Comment:
This change doesn't seem to belong to this commit.

Reply:
It is related. I remove settings->proto/hoss/port from the settings source file.